### DISCUSSION TITLE
Deleting osu!lazer didn't delete the folder containing all the user files

### DISCUSSION THREAD

**Discussion by nuewink** [ppy/osu#29814] [Q&A] [answered]
Hello. I set the custom path to E:\osu-lazer to store all the maps and stuff on my 2nd SSD. I deleted lazer through the control panel "programs and components" tab, and the folder is still there with all of the 75 gigabytes of songs I downloaded.
`Version = 2024.906.2-lazer` (found in "game.ini")
![image](https://github.com/user-attachments/assets/f1d850ed-7153-438f-87a8-e68de3dfa93c)

**Comment by peppy** [comment, marked as answer]
This is intentional.

You can delete the folder if you want to.

**Reply by nuewink** [reply]
thanks!
